.github/workflows/ci.yml

```
@@ -240,3 +240,29 @@ jobs:
           -e ANTHROPIC_API_KEY=test_key \
           novaeval:test \
           python -m pytest tests/ -v --tb=short --junitxml=junit.xml -o junit_family=legacy
+
+  gemini-integration-test:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && secrets.GEMINI_API_KEY != ''
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)